CHANGES.rst

```diff
@@ -29,6 +29,47 @@ vizier
 - Change the type of raised error when the catalog is not found in ``Vizier.get_catalog_metadata``
   from ``IndexError`` to ``EmptyResponseError`` [#2980]
 
+simbad
+^^^^^^
+
+- The ``ROW_LIMIT`` value to have the maximum number of rows is now -1 instead of 0. This
+  allows to retrieve an empty table with the output's meta-data [#2954]
+
+- ``ROW_LIMIT`` can now be set at instantiation (ex: ``simbad = Simbad(ROW_LIMIT=10))``) [#2954]
+
+- ``list_votable_fields`` now return an astropy Table with added fields information instead
+  of a list of strings [#2954]
+
+- ``list_votable_fields`` is now queried directly from SIMBAD instead of reading a file
+  in astroquery. This prevents it from being outdated [#2954]
+
+- ``get_votable_fields`` now prints the table name and column name instead of just the
+  column name [#2954]
+
+- all query methods now accept ``get_adql`` boolean argument that returns the ADQL string
+  instead of sending the request to SIMBAD. The ``verbose`` and ``get_query_payload``
+  are removed from all methods [#2954]
+
+- all query methods except ``query_tap`` and ``query_criteria`` now accept a ``criteria``
+  argument to restrict the results with custom criteria [#2954]
+
+- ``query_objects`` outputs now have an additional column ``typed_id`` that contains 
+  the requested object's name as typed within astroquery. The ``votable_field`` option
+  ``typed_id`` is removed [#2954]
```

Comment:
I don't like the word `typed` here - it implies to me "classification", but I think you mean "user-entered".  Can we instead add this as `user-specified-id` or something more clear and explicit?

Reply:
Sure, I just kept the former name of the votable option. We can switch to your suggestion :)